### ISSUE TITLE
Best way to calculate surface area of Volumes

### ISSUE THREAD

**Issue by h-OUS-e** [facebookresearch/pytorch3d#1136]
Hello,

I want to try and calculate the surface area and the volume of a random Volume that I generate with `pytorch3d.structures.Volumes`. My thought is to convert a the densities in the volume to a mesh using cubify and do the calculations on the mesh with trimesh.
Is there a better way to do this analysis with pytorch3d directly? using the surface area and volume from trimesh library makes the loss not differentiable and becomes hard to do backprop.

Thank you

**Comment by bottler** [comment]
There are many ways to extract surfaces from volumes, and they won't agree on the surface area. This is a research-type question we can't help with.

**Comment by h-OUS-e** [comment]
sorry let me rephrase the question, is there anyway I could extract the surface area of a Volume while keeping it differentiable (so without using external libraries that detach the tensor)?

**Comment by bottler** [comment]
> sorry let me rephrase the question, is there anyway I could extract the surface area of a Volume while keeping it differentiable (so without using external libraries that detach the tensor)?

Not with PyTorch3D. I think this might still be a research question.
